rtfs_cluster/rtfs_cluster_test.go

```diff
@@ -1,22 +1,36 @@
 package rtfs_cluster_test
 
-const testPIN = "QmNZiPk974vDsPmQii3YbrMKfi12KTSNM7XMiYyiea4VYZ"
+import (
+	"fmt"
+	"testing"
+
+	"github.com/RTradeLtd/Temporal/rtfs_cluster"
+)
+
+const (
+	testPIN        = "QmNZiPk974vDsPmQii3YbrMKfi12KTSNM7XMiYyiea4VYZ"
+	nodeOneAPIAddr = "192.168.1.101"
+	nodeTwoAPIAddr = "192.168.2.101"
```

Comment:
did i put this here? (I think i did) - we might want some light documentation regarding why these values are set (because theyre in the test configuration started up by `make testenv`)

Reply:
Nope, you put them in `rtfs_test` so I copied them :D 
I've also updated the comments